network/views.csv

```diff
@@ -0,0 +1,18 @@
+Mount,View,Label,Dip,Azimuth,Description,Start Date,End Date
```

Comment:
So label captures what the camera is seeing, description covers more where the camera is _and_ what it is seeing. From what I see and know this seems quite complete.